src/__tests__/batchUpdates-test.js

```diff
@@ -18,44 +18,40 @@ function todoReducer(state = { todos: [] }, action) {
     : state;
 }
 
-function testBatchedUpdates(createStore) {
-  const store = createStore({ todos: todoReducer });
+function testBatchedUpdates(middleware) {
+  const store = middleware(createStore)(todoReducer);
 
   const renderSpy = spy();
+  const { addTodo } = bindActionCreators(actionCreators, store.dispatch);
+  class TestComponent extends Component {
+    render() {
+      renderSpy();
+      return <div {...this.props} addTodo={addTodo} />;
+    }
+  }
+  const ConnectedComponent = connect(state => {
+    return { todos: state.todos, f: () => {} };
```

Comment:
I think the test failure has something to do with using the new `connect` function rather than the `Connector` component.

If the function `f` is included in the returned state, the case without `batchedUpdates` passes and the one with fails.  Removing `f` from the returned state reverses this such that the `batchedUpdates` case passes and the case without fails.